packages/ui/tsconfig.json

```diff
@@ -8,9 +8,9 @@
     "importHelpers": true,
     "isolatedModules": true,
     "jsx": "react-jsx",
-    "lib": ["es2021.intl", "dom"],
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "lib": ["es6", "es2021.intl", "dom"],
```

Comment:
I'm not completely sure why this wasn't causing errors before, but our `lib` config didn't include a standard library. So I added the `es6` as a reasonable baseline.